### PR TITLE
(WIP) Feat: Add freeze mints to candy machine

### PR DESCRIPTION
**Pull request by stegaBOB**

### CODE REVIEW

File: candy-machine/program/tests/core/master_edition_v2.rs [hunk 1]
```diff
@@ -97,7 +95,11 @@ impl MasterEditionManager {
             )],
             Some(&self.authority.pubkey()),
             &[&self.authority],
-            context.last_blockhash,
+            context
+                .banks_client
+                .clone()
+                .get_new_latest_blockhash(&context.banks_client.get_latest_blockhash().await?)
```
Comment (thlorenz) [inline candy-machine/program/tests/core/master_edition_v2.rs:101]: I'd wrap this in a function. It's repeated all over at this point :)

File: candy-machine/js/src/generated/instructions/index.ts [hunk 2]
```diff
@@ -2,8 +2,11 @@ export * from './addConfigLines';
 export * from './initializeCandyMachine';
 export * from './mintNft';
 export * from './removeCollection';
+export * from './removeFreeze';
 export * from './setCollection';
 export * from './setCollectionDuringMint';
+export * from './setFreeze';
+export * from './thawNft';
```
Comment (thlorenz) [inline candy-machine/js/src/generated/instructions/index.ts:9]: looks like those files aren't added?

File: candy-machine/js/src/generated/accounts/index.ts [hunk 3]
```diff
@@ -1,2 +1,3 @@
 export * from './CandyMachine';
 export * from './CollectionPDA';
+export * from './FreezePDA';
```
Comment (thlorenz) [inline candy-machine/js/src/generated/accounts/index.ts:3]: Same as for instructions .. missing the new account file here.

File: candy-machine/js/src/generated/instructions/index.ts (hunk 2 above)
Comment (stegaBOB) [reply candy-machine/js/src/generated/instructions/index.ts:9]: Whoops!

File: candy-machine/program/tests/core/master_edition_v2.rs (hunk 1 above)
Comment (stegaBOB) [reply candy-machine/program/tests/core/master_edition_v2.rs:101]: Definitely a good idea!

**Comment by thlorenz** [comment]
Wondering what the status on this is (it's still in draft mode).
If you want me to have another closer look please rebase on master.

Also we should probably think about some smoke-test that would exercise the TS SDK to the point where we detect at least that code is invalid and/or that files are missing.

Could be similar to what I do in [solita](https://github.com/metaplex-foundation/solita/blob/master/test/utils/verify-code.ts).

If so I could pull those utils into a separate package so we can use them in here for that.

(Would be a separate PR obviously).

**Comment by anvog777** [comment]
Looking forward for this. Glad to help.

**Comment by stegaBOB** [comment]
Time to do a super fun manual rebase :)

**Comment by samuelvanderwaal** [comment]
@stegaBOB this has been replaced by #669 and can be closed, correct?

**Comment by stegaBOB** [comment]
yup we can close this

**Pull request closed without merging**